openproblems/tasks/dimensionality_reduction/methods/preprocessing.py

```diff
@@ -1,11 +1,15 @@
 from ....tools.normalize import log_cpm
+from anndata import AnnData
 
 import scanpy as sc
 
 
-def preprocess_scanpy(adata):
+def preprocess_logCPM_1kHVG(adata: AnnData) -> AnnData:
```

Comment:
Move to `openproblems.tools.normalize` per #329

Reply:
Done, and (hopefully) updated in all the methods.